src/CommunityVoices/App/Api/View/Image.php

```diff
@@ -109,12 +109,13 @@ protected function getImageNextImage()
 
         return $response;
     }
-    public function postImageUpdate()
+    protected function postImageUpdate()
```

Comment:
Good, but this is still missing a return.

Reply:
done.